rename_labels.json

```diff
@@ -0,0 +1,161 @@
+{
+    "id": "rename_labels",
+    "summary": "Rename dimension labels",
+    "description": "Renames the labels of a dimension in the data cube. Afterwards, the dimension can be referenced with the new name.",
```

Comment:
> Afterwards, the dimension can be referenced with the new name.
 
this sounds a bit like the dimension name itself is/can be changed. I think this second sentence is not really necessary.

Reply:
Too much copy&paste from rename_dimension. Removed the sentence from both processes.